invenio_app_ils/documents/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -354,7 +354,16 @@
               "type": "keyword"
             },
             "publisher": {
-              "type": "keyword"
+              "type": "text",
```

Comment:
this change is quite complicated to introduce in production, read https://www.elastic.co/guide/en/elasticsearch/reference/7.15/indices-put-mapping.html#updating-field-mappings
so lets avoid modifying existing fields